client/src/components/Cards/Tips/index.js

```diff
@@ -0,0 +1,55 @@
+import * as S from './style';
+import * as T from '../../Typography';
+import Icon from '../../Icon';
+
+const colorArray = [
+  {
+    bg: 'neutralSurface',
+    border: 'neutralMid',
+    text: 'neutralMain',
+    icon: 'neutralMid',
+  },
+  {
+    bg: 'secondaryLight',
+    border: 'secondaryMain',
+    text: 'secondaryMain',
+    icon: 'secondaryMain',
+  },
+  {
+    bg: 'secondaryMain',
+    border: 'white',
+    text: 'white',
+    icon: 'white',
+  },
+  {
+    bg: 'primaryLight',
+    border: 'primaryMain',
+    text: 'neutralMain',
+    icon: 'primaryMain',
+  },
+];
+
+// create function that programmatically cycles through the colorArray in order
+const getColor = (index) => {
+  const _index = index % colorArray.length;
+  return colorArray[_index];
+};
+
+const Tips = ({ tips = [], ...props }) => {
+  return (
+    <S.Section {...props}>
+      {tips.map((tip, index) => (
+        <S.Tip key={index} color={getColor(index)} mb="3">
+          <Icon icon="bulb" color={getColor(index).icon} mr="2" />
+          {typeof tip === 'string' ? (
+            <T.H3 color={getColor(index).text}>{tip}</T.H3>
+          ) : (
+            tip
+          )}
+        </S.Tip>
```

Comment:
this component not macing the wireframe, let say if you want to render this tip on the dashboard `https://www.figma.com/file/2b1vIUlRczjuwhWr9cE9OY/Wireframes?node-id=1932%3A9598` so I need to pass to the component an array `[ , , , "Interested in more specific statistics? Get in touch with hydefoundation@
hyde-housing.co.uk"]`
but this will render the first three tip with empty content,
what I would sugget is to keep the structure for the colorArray but change it to object instead 
` colorObj = {
  tip1: ..,
  tip2:..,
},`
this pass with the component to take children and tip 
`<Tips tip="tip1">
children here
</Tips>
`
and then let the colors depend on the tip prop

Reply:
Cool, thanks @fadeomar - I've modified it so the user can feed in a starting color position. That way you can still customise for individual ones, plus then when there's a list, even if you start on a custom colour it would then iterate through them always in the same consistent way, which I think we should do